File: Mozu.SiteBuilder.UX.Admin/Scripts/app/model/PaymentTransaction.js
/**
 * @class Taco.model.PaymentTransaction
 */
Ext.define('Taco.model.PaymentTransaction', {
    extend: 'Taco.core.data.Model',

    fields: [{
        "name": "id",
        "type": "string",
        "useNull": true
    }, {
        "name": "paymentServiceTransactionId",
        "type": "string",
        "useNull": true
    }, {
        "name": "orderId",
        "type": "string",
        "useNull": true
    }, {
        "name": "transactionType",
        "type": "string",
        "useNull": true
    }, {
        "name": "paymentReference",
        "type": "auto",
        "useNull": true
    }, {
        "name": "billingAddress",
        "type": "auto",
        "useNull": true
    }, {
        "name": "status",
        "type": "string",
        "useNull": true
    }, {
        "name": "interactions",
        "type": "auto",
        "useNull": true
    }, {
        "name": "isRecurring",
        "type": "boolean",
        "useNull": true
    }, {
        "name": "createDate",
        "type": "date",
        "useNull": true
    }, {
        "name": "createBy",
        "type": "string",
        "useNull": true
    }, {
        "name": "updateDate",
        "type": "date",
        "useNull": true
    }, {
        "name": "updateBy",
        "type": "string",
        "useNull": true
    }],

    proxy: {
        type: 'ajaxproxy',
        api: {
            read: '/admin/app/order/paymenttransaction/list',
            create: '/admin/app/order/paymenttransaction/create',
            update: '/admin/app/order/paymenttransaction/edit',
            destroy: '/admin/app/order/paymenttransaction/delete'
        },
        reader: {
            type: 'json',
            root: 'items',
            successProperty: 'success',
            messageProperty: "message"
        },
        writer: {
            allowSingle: false,
            type: 'json'
        }
    }
});

File: Mozu.SiteBuilder.UX.Models/Orders/PaymentTransaction.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Mozu.SiteBuilder.UX.Models.Customers;

namespace Mozu.SiteBuilder.UX.Models.Orders
{
    [DataContract]
    public class PaymentTransaction : ModelBase
    {
        [DataMember(Name = "id")]
        public int? Id { get; set; }

        [DataMember(Name = "paymentServiceTransactionId")]
        public string PaymentServiceTransactionId { get; set; }

        [DataMember(Name = "orderId")]
        public string OrderId { get; set; }

        [DataMember(Name = "transactionType")]
        public string TransactionType { get; set; }

        [DataMember(Name = "paymentReference")]
        public PaymentReference PaymentReference { get; set; }

        [DataMember(Name = "billingAddress")]
        public Contact BillingAddress { get; set; }

        [DataMember(Name = "status")]
        public string Status { get; set; }

        [DataMember(Name = "interactions")]
        public List<PaymentTransactionInteraction> Interactions { get; set; }

        [DataMember(Name = "isRecurring")]
        public bool IsRecurring { get; set; }

        [DataMember(Name = "createDate")]
        public DateTime? CreateDate { get; set; }

        [DataMember(Name = "createBy")]
        public string CreateBy { get; set; }

        [DataMember(Name = "updateDate")]
        public DateTime? UpdateDate { get; set; }

        [DataMember(Name = "updateBy")]
        public string UpdateBy { get; set; }
    }
}